src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -410,9 +374,6 @@ def __init__(self, input_directory, target_path, fuzzing_strategies):
     if not list_full_file_paths(self.input_directory):
       write_dummy_file(self.input_directory)
 
-    # Make a new corpus without oversized inputs if needed.
-    self.create_new_if_needed()
```

Comment:
@inferno-chromium What are we going to do about Chromium's AFL (and other users maybe?).
This change will break things there. Are you sure we don't want vanilla AFL support anymore?